app/controllers/api/users_controller.rb

```diff
@@ -4,9 +4,36 @@ module Api
   class UsersController < ApplicationController
     include AccessTokenVerifiable
     before_action :validate_user_id, only: %i[show]
+
     # GET /api/users/:id
     def show
-      render_user user
+      render :show, status: :ok, locals: { user: }
+    end
+
+    # POST /api/users
+    def create
+      name = params[:name]
+      email = params[:email]
+      password = params[:password]
+
+      create_user = User.create(
+        name:,
+        email:,
+        password:,
+        password_confirmation: password,
+        activated:             false
+      )
+
+      status =
+        if !create_user.invalid?
```

Comment:
ここはなんだか不思議な書き方になってますね。not invalid なので、単に valid? という確認で良さそうです。
```suggestion
        if create_user.valid?
```